### PR TITLE
Revised Documentation for Accuracy

### PR DESCRIPTION
**Pull request by dmccoystephenson**
## Problem
The README is outdated and disorganized.

## Solution
The README has been revised for accuracy & clarity.

## Dependent PR
This PR includes changes introduced in #10 and should be merged after.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,29 @@
 
 # jpo-security-svcs
-This module exposes a RESTful API for performing cryptographic functions. The following paths identify the functions:
-|Verb|path|Content Type|Functionality|Request Body Format|Response Body Format|
-|--|--|--|--|--|--|
-|POST|/sign|application/json|signs data provided in the body of the request|{"message":"Base64 encoded unsigned data"}|{"result": "Base64 Encoded Signed Data"}
+This module exposes a RESTful API for performing cryptographic signing of messages.
 
+Cryptographic signing is carried out against a remote instance, which requires mutual TLS authentication to secure the communications. This requires using private and public keys to secure the communications between this instance and the remote signing instance.
```
Comment (drewjj) [inline README.md:5]: If you want to make the repo sound real fancy, you can call it TLS using asymmetric cryptography to secure instance communication. Nothing wrong with explaining how it works, though.

**Review by drewjj — APPROVED** [review]
Looks great!

**Review by payneBrandon — APPROVED** [review]
lgtm!

File: README.md [hunk 2]
```diff
@@ -81,16 +40,62 @@ Expected output:
...
+| -------- | ------- | ------------- | ------------------------------- |
+| server.port | The port number to which this service will be listening. | 8090 | SERVER_PORT |
+| sec.cryptoServiceBaseUri | Cryptographic service endpoint URI excluding path. For example, `http://<ip>:<port>` OR `http://server.dns.name` including the port number, if any. | - | SEC_CRYPTO_SERVICE_BASE_URI|
+| sec.cryptoServiceEndpointSignPath | The REST endpoint path of the external service. | /tmc/signtim |SEC_CRYPTO_SERVICE_ENDPOINT_SIGN_PATH |
 | sec.useCertificates | Whether to use certificates or not. | true | SEC_USE_CERTIFICATES |
 | sec.keyStorePath | The path to the keystore file. | /home/cert.jks | SEC_KEY_STORE_PATH |
 | sec.keyStorePassword | The password for the keystore file. | password | SEC_KEY_STORE_PASSWORD |
+
+## Mutual TLS Authentication
+For enhanced security, MTLS is used to communicate with a remote signing system. This requires properly configured certificates from the remote system to property perform - work with your remote signing authority to obtain necessary certificates.
```
Comment (mwodahl) [inline README.md]: Should this be 'properly perform' instead of 'property perform'?

Comment (dmccoystephenson) [reply README.md]: Yes, thanks for catching that! This has been fixed!

**Review by Michael7371 — APPROVED** [review]
Looks good!

**Review by mwodahl — APPROVED** [review]

**Pull request merged**